core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/util/deprecationMessages.kt

```diff
@@ -213,4 +213,10 @@ internal const val ALL_COLS_EXCEPT_REPLACE = "this.allColsExcept { other }"
 internal const val ALL_COLS_EXCEPT_REPLACE_VARARG = "this.allColsExcept { others.toColumnSet() }"
 internal const val EXCEPT_REPLACE = "this.except { other }"
 internal const val EXCEPT_REPLACE_VARARG = "this.except { others.toColumnSet() }"
+
+internal const val COLS_TO_ALL = "This `cols()` overload will be removed in favor of `all()`. $MESSAGE_1_0"
```

Comment:
You added it in the `// region keep across releases` but with a message for 1.0. How do you want to deprecate this?